src/components/Profile/ReviewAndFollow.tsx

```diff
@@ -0,0 +1,28 @@
+import { UserDataProps } from '@/types';
+
+type ReviewAndFollowProps = {
+  user: UserDataProps;
+};
+
+const ReviewAndFollow = ({ user }: ReviewAndFollowProps) => {
+  return (
+    <div className="mt-10">
+      <ul className="flex justify-around text-center min-w-[300px]">
+        <li className="cursor-pointer">
+          <div>게시물</div>
+          <div>{user.posts.length || 0}</div>
```

Comment:
```suggestion
          <div>{user.posts.length}</div>
```
이렇게 써도 될 거 같은데 어떤가용?

Reply:
저거 없으면 오류 나더라
user 데이터 체크 다시해볼게유